lib/mongoid_rails_migrations/active_record_ext/migrations.rb

```diff
@@ -430,10 +474,10 @@ def record_version_state_after_migrating(version)
         # end
         if down?
           @migrated_versions.delete(version)
-          DataMigration.where(:version => version.to_s).first.destroy
+          DataMigration.where(:version => version.to_s).destroy_all
         else
           @migrated_versions.push(version).sort!
-          DataMigration.create(:version => version.to_s)
+          DataMigration.find_or_create_by(:version => version.to_s)
```

Comment:
This change and the one line `477` prevent multiples entries with the same version to be written in the `data_migrations` collection.

Reply:
FWICS in code yes but I found in one of our database that sometimes it seems not working as expected:
```
> DataMigration.where(version: "20170809160116").count
=> 6
```
Not sure of why, the migration probably failed multiple times but when it failed, it should haven't added multiple entries in the `data_migrations` collection.

As the same version cannot be used as multiple migrations, we're safe to add this IMO.

Reply:
Why not ^^